integration/spark/src/main/scala/org/apache/spark/sql/catalyst/CarbonParserUtil.scala

```diff
@@ -1135,11 +1135,7 @@ object CarbonParserUtil {
         }
         DataTypeInfo("decimal", precision, scale)
       case _ =>
-        if (isColumnRename) {
-          DataTypeInfo(DataTypeConverterUtil.convertToCarbonType(dataType).getName.toLowerCase)
-        } else {
-          throw new MalformedCarbonCommandException("Data type provided is invalid.")
-        }
+        DataTypeInfo(DataTypeConverterUtil.convertToCarbonType(dataType).getName.toLowerCase)
```

Comment:
is the parameter isColumnRename of this method needed?

Reply:
Yes. useless to throw MalformedCarbonCommandException("Data type provided is invalid.")
CarbonAlterTableColumnRenameCommand.validateColumnDataType will validate the given datatype